os/arch/arm/src/amebad/amebad_spi.c

```diff
@@ -1089,10 +1117,26 @@ static void amebad_spi_bus_initialize(struct amebad_spidev_s *priv)
 	DEBUGASSERT(&priv->spi_object);
 
 	priv->spi_object.spi_idx = priv->spi_idx;
-	spi_init(&priv->spi_object, priv->spi_mosi, priv->spi_miso, priv->spi_sclk, priv->spi_cs);
+	spi_init(&priv->spi_object, priv->spi_mosi, priv->spi_miso, priv->spi_sclk, priv->spi_cs0);
 
 	spi_format(&priv->spi_object, priv->nbits, priv->mode, priv->role);
 
+	/* Initialize the SPI semaphore that enforces mutually exclusive access */
```

Comment:
Hi, I understand that I will reverting the spics0 & spics1 back to the initial setup. As for the multi-slaves configuration, shall I stick to using gpio_cs setup instead? I will separate using preprocessor conditions for single and multi in this regard.
 
Kai

Reply:
you added CONFIG_SPI_CS, and it disabled as a default value. hence we can accept it.